test/data/commands.txt

```diff
@@ -1 +1 @@
-if [ `uname` = 'Darwin' ]; then export SD_CURL_CMD_WRAPPER="eval"; else export SD_CURL_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 7.54.1 core/curl')"; fi && cd $SD_ARTIFACTS_DIR && find . -type f -print > manifest.txt && find . -type f -exec sh -c 'f=`echo $1 | sed "s/^\.\///"`; encoded=$($SD_CURL_CMD_WRAPPER "curl -s -w \"%{url_effective}\" --data-urlencode \"$f\" -G \"\" | sed -e \"s/^\/?//\""); $SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" -H \"Content-Type: text/plain\" -X PUT https://store.screwdriver.cd/v1/builds/$SD_BUILD_ID/ARTIFACTS/$encoded -T ./\"$f\""' -- {} \;
+if [ `uname` = 'Darwin' ]; then &&     export SD_CURL_CMD_WRAPPER="eval" &&     export SD_ZIP_CMD_WRAPPER="" && else &&     export SD_CURL_CMD_WRAPPER="$(echo 'sd-step exec --pkg-version 7.54.1 core/curl')" &&     export SD_ZIP_CMD_WRAPPER="$(echo 'sd-step exec core/zip')" && fi &&  && cd $SD_ARTIFACTS_DIR && find . -type f -print > manifest.txt &&  && if [ "$SD_ZIP_ARTIFACTS" = "true" ]; then &&     ZIPFILE=SD_ARTIFACT.zip &&     find . -type f -exec $SD_ZIP_CMD_WRAPPER zip $ZIPFILE {} + &&     f=`echo $ZIPFILE | sed "s/^\.\///"` &&     encoded=$($SD_CURL_CMD_WRAPPER "curl -s -w \"%{url_effective}\" --data-urlencode \"$f\" -G \"\" | sed -e \"s/^\/?//\"") &&     $SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" -H \"Content-Type: text/plain\" -X PUT https://store.screwdriver.cd/v1/builds/$SD_BUILD_ID/ARTIFACTS/$encoded -T ./\"$f\"" && else &&     find . -type f -exec sh -c 'f=`echo $1 | sed "s/^\.\///"`; encoded=$($SD_CURL_CMD_WRAPPER "curl -s -w \"%{url_effective}\" --data-urlencode \"$f\" -G \"\" | sed -e \"s/^\/?//\""); $SD_CURL_CMD_WRAPPER "curl -H \"Authorization: Bearer $SD_TOKEN\" -H \"Content-Type: text/plain\" -X PUT $STORE_URL/v1/builds/$SD_BUILD_ID/$ARTIFACTS_DIR_SUFFIX/$encoded -T ./\"$f\""' -- {} \; && fi
```

Comment:
Does the script work? It's doing `then &&  export xxx`.....

Reply:
fixed